editors/vscode/package.json

```diff
@@ -387,7 +387,7 @@
           "scope": "resource",
           "type": "array",
           "default": [],
-          "description": "Array of Taplo rules in JSON format, see https://taplo.tamasfe.dev/configuration/file.html#rules. The rules given here are appended to existing rules from config files, if any. There is no conversion done, all object keys must be snake_case, including formatting rules."
+          "description": "Array of Taplo rules in JSON format, see [Configuration File - Rules](https://taplo.tamasfe.dev/configuration/file.html#rules). The rules given here are appended to existing rules from config files, if any. There is no conversion done, all object keys must be snake_case, including formatting rules."
```

Comment:
You can also wrap a URL between `<` and `>` to make it clickable.

Reply:
I considered that but the link is really long so I thought a title was best. Any preference?

Reply:
I don't know VSCode. If the link is always clickable then keeping it like that is fine and indeed more readable. However if it's possible that for some reason it's not clickable, then it's better to use `<>` for users to copy/paste it.

Reply:
Yeah it should be clickable with the `[]()` syntax.